.github/workflows/backport.yml

```diff
@@ -0,0 +1,38 @@
+name: Backport
+on:
+  workflow_dispatch:
+    inputs:
+      number:
+        description: "The pull request # to backport"
+        required: true
+
+jobs:
+  backport:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          # history is needed to run git cherry-pick below
+          fetch-depth: 0
+
+      - name: Set git user
+        run: .github/scripts/set-git-user.sh
+
+      - name: Create pull request
```

Comment:
This creates a pull request against release branches?

Reply:
ya, you select the release branch when you run the workflow. I just added a check above this to verify that it's not being run against `main`.